### DISCUSSION TITLE
OPENSOURCE

### DISCUSSION THREAD

**Discussion by rashidiqbal007** [lllyasviel/LuminaBrush#1] [General]
Any plans to opensource? And when!!!

**Comment by icongirl** [comment]
After using the uniform lighting plugin, the text on the product packaging deformed. Can we solve the problem of text distortion on product packaging? The text of the product needs to be evenly polished without any impact.

**Comment by icongirl** [comment]
In the e-commerce industry scene, this evenly lit model is particularly awesome and easy to use.Good!!
